misc/deb822/materialize.sources

```diff
@@ -0,0 +1,55 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+Types: deb
+URIs: http://apt.materialize.com/
+Suites: generic
+Components: main
+Architectures: amd64
```

Comment:
Not sure what the impact of this is, but we support arm too. Can you just leave this field out, like you can in the old-style format?

Reply:
Yup, removed.